sys/include/suit.h

```diff
@@ -148,7 +189,7 @@ typedef struct {
     /** List of components in the manifest */
     suit_component_t components[SUIT_COMPONENT_MAX];
     unsigned components_len;        /**< Current number of components */
-    uint32_t component_current;     /**< Current component index */
+    uint8_t component_current;      /**< Current component index */
```

Comment:
Are components caped at 256?

Reply:
Yup, do you want me to add a compile-time check for that?

I find it very unlikely that we every going to create a manifest that updates more than 256 different binary objects on a single update run.

Reply:
No I think its fine.